shopfloor_base/__manifest__.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2020 Camptocamp SA (http://www.camptocamp.com)
+# Copyright 2020 Akretion (http://www.akretion.com)
+# Copyright 2020 BCIM
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Shopfloor Base",
+    "summary": "Core module for creating mobile apps",
+    "version": "13.0.1.0.0",
+    "development_status": "Alpha",
+    "category": "Inventory",
+    "website": "https://github.com/OCA/wms",
+    "author": "Camptocamp, BCIM, Akretion, Odoo Community Association (OCA)",
+    "maintainers": ["guewen", "simahawk", "sebalix"],
+    "license": "AGPL-3",
+    "application": True,
+    "depends": [
+        "base_jsonify",
+        "base_rest",
+        "rest_log",
```

Comment:
Why do we have a dependency on `rest_log`? The only motivation seems to put some entries into the odoo menu :thinking:

Reply:
well, is not required indeed. Maybe for now I can move it to `shopfloor` as we expect it to be there. Other long term option is having a dumb module `shopfloor_rest_log` but it seems too much for just having a menu item.